### PR TITLE
improve "Open With" functionality after user feedback

### PR DESCRIPTION
**Pull request by HubertHuckevoll**

### CODE REVIEW

File: Library/FMTools/OpenWith/openwith.rev
Comment (bluewaysw) [inline Library/FMTools/OpenWith/openwith.rev:1]: Yes, here for the release, starting with 6.0.0.0 would be consistent.

**Review by bluewaysw — CHANGES_REQUESTED** [review]

File: Tools/build/product/bbxensem/Template/geos.ini [hunk 1]
```diff
@@ -89,6 +89,13 @@ filenameTokens = {
   *.* = "FILE",0,"TeEd",0
 }
 
+openWith = {
+Desk Accessories\\EC-long(Text File Editor)
+Extras\\EC-long(Graphics Viewer)
+\\EC-long(WebMagick)
+Extras\\EC-long(WebBox)
+}
```
[the remark this answers is not in the document]
Comment (HubertHuckevoll) [reply Tools/build/product/bbxensem/Template/geos.ini:97]: You are right in that I didn't think of this at all, but in the current german builds, Graphics Viewer is Graphics Viewer, WebBox is WebBox and WebMagick is WebMagick... so, no fixes needed...?

File: Installed/Library/FMTools/OpenWith/openwith
[the remark this answers is not in the document]
Comment (HubertHuckevoll) [reply Installed/Library/FMTools/OpenWith/openwith:1]: I 've added this as a recommendation of RainerB, but I don't think I fully understand it's purpose yet ;-)

Comment (bluewaysw) [inline Installed/Library/FMTools/OpenWith/openwith:1]: What this file is about?

**Review by bluewaysw — CHANGES_REQUESTED** [review]
Please remove files that are unused.

File: Installed/Library/FMTools/OpenWith/openwith
Comment (HubertHuckevoll) [reply Installed/Library/FMTools/OpenWith/openwith:1]: File removed.

**Review by bluewaysw — APPROVED** [review]

**Pull request merged**